packages/website/src/examples/layout/Table/Table.demo.tsx

```diff
@@ -1,39 +1,148 @@
+
+interface IExampleRowData {
+    userId: number;
+    id: number;
+    title: string;
+    body: string;
+}
 
 export default () => {
     const columnHelper = createColumnHelper<IExampleRowData>();
     const columns: Array<ColumnDef<IExampleRowData>> = [
-        columnHelper.accessor('username', {
-            header: 'Name',
-            cell: (info) => info.row.original.username,
-            enableSorting: false,
+        columnHelper.accessor('userId', {
+            header: 'User ID',
+            cell: (info) => info.row.original.userId,
         }),
-        columnHelper.accessor('city', {
-            header: 'City',
-            cell: (info) => info.row.original.city,
-            enableSorting: false,
+        columnHelper.accessor('id', {
+            header: 'Post ID',
+            cell: (info) => info.row.original.id,
         }),
-        columnHelper.accessor('password', {
-            header: 'Password',
-            cell: (info) => info.row.original.password,
-            enableSorting: false,
+        columnHelper.accessor('title', {
+            header: 'Title',
+            cell: (info) => info.row.original.title,
         }),
+        Table.CollapsibleColumn as ColumnDef<IExampleRowData>,
     ];
-    return <Table data={dataForRows} columns={columns} />;
+    const [data, setData] = useState([]);
+    const [loading, setLoading] = useState(false);
+    const [pages, setPages] = useState(1);
+
+    const fetchData = (state: any) => {
+        setLoading(true);
+        const searchParams = new URLSearchParams({
+            _sort: state.sorting?.[0]?.id ? state.sorting?.[0]?.id : 'userId',
+            _order: state.sorting?.[0]?.desc ? 'desc' : 'asc',
+            _page: state.pagination.pageIndex ? state.pagination.pageIndex + 1 : 1,
+            _limit: state.pagination.pageSize,
+            userId: state.predicates.user,
+            title_like: state.globalFilter,
+        });
+        if (state.predicates.user === '') {
+            searchParams.delete('userId');
+        }
+        if (!state.globalFilter) {
+            searchParams.delete('title_like');
+        }
+        fetch(`https://jsonplaceholder.typicode.com/posts?${searchParams.toString()}`)
+            .then((response) => response.json())
+            .then((json) => setData(json))
+            .then(() => setPages(Math.ceil(100 / state.pagination?.pageSize)))
+            .catch((e) => console.log(e));
+        setLoading(false);
+    };
+
+    return (
+        <Table
+            data={data}
+            columns={columns}
+            noDataChildren={<NoData />}
+            onMount={(state) => {
+                fetchData(state);
+            }}
+            onChange={(state) => {
+                fetchData(state);
+            }}
+            loading={loading}
+            initialState={{dateRange: [previousDay, today], predicates: {user: ''}}}
+            getExpandChildren={(datum) => <div className="py2">{datum.body}</div>}
```

Comment:
I think we should avoid any css class coming from `plasma-style` we probably could use just a box with prop `py="md"` or something along those lines

Reply:
noted will change it